GitHub/GitHubCLI.pkg.recipe

```diff
@@ -83,7 +93,7 @@
 							<key>mode</key>
 							<string>0755</string>
 							<key>path</key>
-							<string>usr/local/bin/gh</string>
+							<string>usr</string>
```

Comment:
Just checking — would this `chmod` the entire /usr directory recursively, including items not installed by the GitHubCLI package?

Reply:
Pretty sure this is handed through to [autopkgserver/packager.py](https://github.com/autopkg/autopkg/blob/af858c033643089b8ecbccf0a8ad691d4c28a289/Code/autopkgserver/packager.py#L296) for making sure the contents of the pkgroot have proper perms, meaning it only operates on the payload and otherwise doesn't alter/affect anything else on a target/destination

Reply:
That's sensible. Thank you!